### DISCUSSION TITLE
problem in process my application button

### DISCUSSION THREAD

**Discussion by EnAnsari** [community/community#63645] [GitHub Education] [answered]
### Hi everyone,

Hello, unfortunately, when I upload my student card (using camera capture - I don't use local file), I am redirected to the upload page after clicking the process button of the application. what is the problem? So far, I tried several times with Chrome, Firefox, and even with mobile, but the problem was not solved.
![image](https://github.com/community/community/assets/109433485/f16e192c-8cdd-4f1c-b301-fbdd5e39f6ac)
![image](https://github.com/community/community/assets/109433485/6cbca28a-de74-4fdf-abb5-1f8ec53b4e6f)

**Comment by safnimj** [comment]
First capture that with your system Camera and Upload it

**Reply by GhostOf0days** [reply]
Please read the post thoroughly before answering.

**Comment by advayc** [comment]
Try taking a picture from your phone and uploading it onto your computer then upload using the file. Afterwards
if that doesn't work, try to convert the image into a JPEG using this website [https://convertio.co/png-jpeg/] then upload.

**Comment by EnAnsari** [comment, marked as answer]
This was a bug from GitHub that was fixed by canceling the process and restarting it.

**Reply by iuu6** [reply]
I have an identical problem, please assist me with it.

**Reply by EnAnsari** [reply]
In my opinion, cancel the registration process and try again later or even at the same time.
After encountering this problem, I started the registration again and this time I was able to upload the document, but I still had problems sending it.
The next day, by unregistering and re-registering again, my problem was solved.

**Reply by iuu6** [reply]
Thanks very much, I try to upload this image today, now it's success.
![图片](https://github.com/community/community/assets/60139353/34a0661d-35e6-4d97-80e2-c71a838e8ce4)
